Makefile

```diff
@@ -0,0 +1,24 @@
+.PHONY: *
+
+all: build-base build-env 
+
+build-base:
+	git clone --depth 1 https://github.com/sunil-sahoo/opencart.git -b docker-deploy /project
+	docker build -t build-env -f /project/Dockerfile.build /project
+
+build-env:
+	docker run -it --rm --name ubuntu-build-env -v /var/run/docker.sock:/var/run/docker.sock -v /project:/project build-env
```

Comment:
You shouldn't have it in this file. Makefile must be only for building and setting up stuff before running compose